account_financial_report/report/general_ledger_xlsx.py

```diff
@@ -125,6 +138,18 @@ def _generate_report_content(self, workbook, report):
             # 2 lines break
             self.row_pos += 2
 
+    def write_initial_balance(self, my_object, type_object):
```

Comment:
It could be nice to perform this check on the object itself, instead of passing the string parameter.
You can try something like `my_object._model == 'res.partner'`